docs/book/starter-guide/pipelines/fetching-pipelines.md

```diff
@@ -141,30 +141,55 @@ zenml pipeline runs list -p <MY_PIPELINE_NAME_OR_ID>
 Each run has a collection of useful metadata which you can access to ensure all runs 
 are reproducible:
 
-#### git_sha
+#### Git SHA
 The Git commit SHA that the pipeline run was performed on. This will only be set 
```

Comment:
```suggestion
The [Git commit SHA](https://www.mikestreety.co.uk/blog/the-git-commit-hash/) that the pipeline run was performed on. This will only be set 
```